.github/workflows/ci.yml

```
@@ -46,6 +46,33 @@ jobs:
           ./sw/tests/run.py sw/tests/run.yaml --simulator verilator
           ./sw/apps/run.py sw/apps/run.yaml --simulator verilator
 
+  sw-snax-cluster-vlt:
+    name: Simulate SW on SNAX Cluster w/ Verilator
+    runs-on: ubuntu-22.04
+    container:
+      image: ghcr.io/pulp-platform/snitch_cluster:ipc-verification
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - name: Load Architecture
+        # The build makefile looks for lru.hjson
+        # Need to copy-paste default or snax config to lru.hjson
+        run: |
+          cp target/snitch_cluster/cfg/snax.hjson \
+          target/snitch_cluster/cfg/lru.hjson
+      - name: Build Hardware
+        run: |
+          make -C target/snitch_cluster bin/snitch_cluster.vlt
+      - name: Build Software
+        run: |
+          make -C target/snitch_cluster sw
```

```suggestion
      - name: Build Hardware
        run: |
          make CFG_OVERRIDE=target/snitch_cluster/cfg/snax.hjson -C target/snitch_cluster bin/snitch_cluster.vlt
      - name: Build Software
        run: |
          make -C target/snitch_cluster sw
```
This is what we use in the Dockerfile in snax-mlir. I know that it essentially does the same as the copy, but it seems just slightly more robust to use the "more abstract API" here and to keep things the same. 
The `lru.hjson` should also used in the later job for the software in this way.

---

OK! I shall test this wait.